packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/corda-v5-flow.test.ts

```diff
@@ -0,0 +1,350 @@
+    const sshConfig = await cordaV5TestLedger.getSshConfig();
+
+    plugin = new PluginLedgerConnectorCorda({
+      instanceId: uuidv4(),
+      sshConfigAdminShell: sshConfig,
+      corDappsDir: "",
+      logLevel,
+      cordaVersion: CordaVersion.CORDA_V5,
+      apiUrl: "https://127.0.0.1:8888",
+    });
+    const listenOptions: IListenOptions = {
+      hostname: "127.0.0.1",
+      port: 0,
+      server,
+    };
+    const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+    const { address, port } = addressInfo;
+    const apiHost = `http://${address}:${port}`;
+    const config = new Configuration({ basePath: apiHost });
+    await plugin.registerWebServices(expressApp);
+    apiClient = new DefaultApi(config);
+    await cordaV5TestLedger.start();
+    expect(cordaV5TestLedger).toBeTruthy();
+  });
+  afterAll(async () => {
+    await cordaV5TestLedger.stop();
+    await cordaV5TestLedger.destroy();
+    await Servers.shutdown(server);
+  });
+  let shortHashID: string;
+  it("Get container", async () => {
+    const container = cordaV5TestLedger.getContainer();
+    const cmd = ["./gradlew", "listVNodes"];
+    const timeout = 180000; // 3 minutes
+    const cwd = "/CSDE-cordapp-template-kotlin";
+    shortHashID = await Containers.exec(container, cmd, timeout, logLevel, cwd);
+  });
+
+  describe("Endpoint Testing", () => {
+    let shortHashAlice = "";
+    let shortHashBob = "";
+    let shortHashCharlie = "";
+    let shortHashDave = "";
+    it("Extract short hash for Alice", () => {
+      shortHashAlice = extractShortHash(shortHashID, "Alice");
+      expect(shortHashAlice).toBeTruthy();
+      expect(`Short hash ID for Alice: ${shortHashAlice}`).toMatch(
+        /Short hash ID for Alice:/,
+      );
+      console.log(`Short hash ID for Alice: ${shortHashAlice}`);
+    });
+    it("Extract short hash for Bob", () => {
+      shortHashBob = extractShortHash(shortHashID, "Bob");
+      expect(shortHashBob).toBeTruthy();
+      expect(`Short hash ID for Bob: ${shortHashBob}`).toMatch(
+        /Short hash ID for Bob:/,
+      );
+      console.log(`Short hash ID for Bob: ${shortHashBob}`);
+    });
+    it("Extract short hash for Charlie", () => {
+      shortHashCharlie = extractShortHash(shortHashID, "Charlie");
+      expect(typeof shortHashCharlie === "string").toBe(true);
+      expect(shortHashCharlie).toBeTruthy();
+      expect(`Short hash ID for Charlie: ${shortHashCharlie}`).toMatch(
+        /Short hash ID for Charlie:/,
+      );
+      console.log(`Short hash ID for Charlie: ${shortHashCharlie}`);
+    });
+    it("Extract short hash for Dave", () => {
+      shortHashDave = extractShortHash(shortHashID, "Dave");
+      expect(shortHashDave).toBeTruthy();
+      expect(`Short hash ID for Dave: ${shortHashDave}`).toMatch(
+        /Short hash ID for Dave:/,
+      );
+      console.log(`Short hash ID for Dave: ${shortHashDave}`);
+    });
+
+    it("CPI test", async () => {
```

Comment:
Please rename the description of the test case to match the endpoint. For example, `listing CPIs`